lib/modable.rb

```diff
@@ -1,9 +1,12 @@
 module Modable
```

Comment:
Great use of module here, can you brainstorm with me later to see if we could implement them elsewhere as well?